### DISCUSSION TITLE
Novità settembre  nuovo anno scolastico

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#535] [Idee e proposte]
Buongiorno ,potrei chiedere quali sono le novità/funzioni che ormai chiediamo/desideriamo da anni verranno introdotte nel registro a partire da settembre del nuovo anno scolastico ?Grazie

**Comment by trinko** [comment]
Francamente non lo so.
Posso solo dire che prossimamente mi dovrò occupare del miglioramento ed eventuale correzione di errori relativamente alle procedure: 
- scrutini con giudizio sospeso
- archiviazione
- passaggio d'anno
